lib/kino/process.ex

```diff
@@ -224,6 +225,14 @@ defmodule Kino.Process do
   target argument can either be a single PID, a list of PIDs, or the atom `:all`
   depending on what messages you would like to retain in your trace.
 
+  ## Options
+
+    * `:label_function` - A function to help label message events. If
```

Comment:
Can we call it `:message_label`? We usually don't append the type to the option names.

I also don't think we should expose `label_from_value`. If you pass a custom function, you usually know better which values you are going to handle, no?

Reply:
> Can we call it :message_label? We usually don't append the type to the option names.

Sure, let me change that now.

> I also don't think we should expose label_from_value. If you pass a custom function, you usually know better which values you are going to handle, no?

Most likely, I just wanted to preserve the initial message, as the code I'll be running it on just wraps the normal `GenServer.call` and `GenServer.cast` with some extra information to the given `GenServer`, meaning the default behavior is close to what I wanted but not quite.

Reply:
Both complaints should be addressed now on the latest push